src/System.Private.CoreLib/src/Internal/Runtime/Augments/RuntimeAugments.cs

```diff
@@ -33,6 +33,12 @@
 
 using Volatile = System.Threading.Volatile;
 
+#if BIT64
```

Comment:
Having `nuint` in public method signatures tends to be problematic because of it makes the public surface vary between different architectures.

Reply:
Noted. What do you suggest?

Reply:
- Add `static ref byte GetSzArrayElementAddress(Array array, int index)` instead of these methods?

Reply:
This has been done